src-plugins/mscAlgorithmPaint/mscAlgorithmPaintToolBox.cpp

```diff
@@ -399,6 +399,7 @@ AlgorithmPaintToolBox::AlgorithmPaintToolBox(QWidget *parent ) :
 
     m_applyButton = new QPushButton( tr("Save") );
     m_applyButton->setToolTip(tr("Save result to the Database"));
+    m_applyButton->setObjectName("saveButton");
```

Comment:
I hate to insist on this, but the other buttons in the same toolbox (that I did not name myself), follow the setObjectName("fooButton") rule, not to mention that two of us prefer that scheme.